.env.docker.scaleway.example

```
@@ -0,0 +1,172 @@
+AI_GITHUB_USER=your-github-user
+AI_GITHUB_PAT=your-github-pat
+
+LANGCHAIN_TRACING_V2=false
+LANGCHAIN_ENDPOINT=https://api.smith.langchain.com
+LANGCHAIN_API_KEY=your-langchain-key
+LANGCHAIN_PROJECT=guidance-engine
+
+
+VECTOR_DB_HOST=chromadb
+VECTOR_DB_PORT=8000
+VECTOR_DB_CREDENTIALS=username:password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VECTOR_DB_CREDENTIALS key should go before the VECTOR_DB_HOST key

(UnorderedKey)